src/sources/http.rs

```diff
@@ -72,6 +72,15 @@ where
         "https://api.ipify.org",
         "https://ifconfig.me/ip",
         "https://ipinfo.io/ip",
+        "https://ifconfig.co/ip",
+        "https://myip.swishmail.com/ip",
+        "https://ipconfig.pw/ip",
+        "https://myipaddr.info/ip",
+        "https://ip.wookingwoo.com/ip",
```

Comment:
I would avoid adding this one to the default as it looks more like a private source

Reply:
It links to a subdomain of someone's blog, adding it here can add traffic to them without really having asked them